### PR TITLE
[WIP] feature: custom text enabled plugin representation (documentation)

### PR DESCRIPTION
**Pull request by ollb**
This is a proposal for the issue #3437 , it would be nice to have it in 3.1. Feel free to comment.
If it seems ok, I'll continue and wrote all necessary code and then I'll provide an implementation in djangocms-text-ckeditor and main text enabled plugins.

### CODE REVIEW

File: docs/reference/plugins.rst [hunk 1]
```diff
@@ -301,6 +301,44 @@ for `djangocms-text-ckeditor <https://github.com/divio/djangocms-text-ckeditor/>
...
+
+When a plugin can be inserted as a child in a text plugin, this method provides
+a way to choose the tag used to encapsulate the plugin representation.
+
+:meth:`in_text_editor_tag` takes 1 arguments:
+
+* ``instance``: The instance of the plugin model
+* ``editor_name``: The name of the parent plugin
+
+This method must return a string. Default implementation returns ``'div'``.
```
Comment (evildmp) [inline docs/reference/plugins.rst]: Does it really return:
    'div'

or does it return:

    div

(i.e. with or without the single quotes)?

Comment (ollb) [inline docs/reference/plugins.rst]: In the documentation when we are refering to the python boolean true, we use ``True``. When refering to a to python string, we might want to use ``'whatever string'`` in order to show the value is a string. That said, I'm fine with ``whatever string``.

Comment (evildmp) [inline docs/reference/plugins.rst]: OK, I think that ``whatever string`` should be preferred, because we already know it's a string.

**Comment by ollb** [comment]
We would love to work on that issue but we are waiting for comments.

**Comment by FinalAngel** [comment]
@evildmp can you provide additional feedback here? @ollb sorry for the long delay :( are you still able to work on it?

**Pull request closed without merging**